docs/rackhd/event_notification.rst

```diff
@@ -1,9 +1,9 @@
 Event Notification
 ------------------
 
-RackHD support event notification via both AMQP and web hook.
+RackHD supports event notification via both web hook and AMQP.
 
-A web hook allows an application to subscribe certain RackHD published events by configured URL, when one of the subscribed events is triggered, RackHD will send a HTTP POST request with event payload to configured URL.
+A web hook allows user to subscribe certain RackHD published events by configured URL, when one of the subscribed events is triggered, RackHD will send a POST request with event payload to configured URL.
```

Comment:
@pengz1 please change user back to application, consumed by applications instead of user is more accurate.